python/pyspark/errors/error_classes.py

```diff
@@ -164,6 +164,11 @@
       "Remote client cannot create a SparkContext. Create SparkSession instead."
     ]
   },
+  "DIFFERENT_PANDAS_DATAFRAME" : {
+    "message" : [
+      "<msg>"
```

Comment:
We should avoid put all error message into single parameter.

But it's fine for now if we don't forget refining this in follow-ups.